### DISCUSSION TITLE
UE5项目中使用puerts，如何重新生成全部js文件

### DISCUSSION THREAD

**Discussion by speedeny** [Tencent/puerts#1811] [Q&A]
1. Puerts成功集成到使用UE5的项目中，用ts写了一部分逻辑，能够正常生成js文件，且执行成功
2. 手动删除Content/Javascript目录下的某个js文件，引擎中调用命令行（puerts.gen full）后，之前被删除的js文件并未生成
3. 用vscode打开删除的js对应的ts文件，任意修改一行保存后，才能正确生成该js文件
请问，如何不通过改ts代码的情况下，引擎中调用命令生成全部的js文件呢？

**Comment by chexiongsheng** [comment]
标准的ts工程，可以用编译ts工程的方式编译。比如tsc，比如ide
